README.md

```diff
@@ -28,6 +28,63 @@ We recommend using MacOS to work with the EDB Docs application.
 
 1. Create a `.env` file: `cp env .env.development`.
 
+At this point you have a couple options.
+- [**Quick set up with Docker**](#get-started-quickly-with-docker)
+
+   This is the preferred set up method. Choose this option if you just want to make updates to documentation, and don't want to worry about installing and managing the correct version of Node.
+
+- [**Set up a full development environment**](#set-up-a-full-development-environment)
+
+   Choose this option if you are an advanced user and need to make more in depth changes to the docs application, such as new functionality.
+
+### Get Started Quickly With Docker
+
+1. Install Docker on your mac. [Follow the direction below.](#install-docker-using-homebrew)
+
+1. If you're a member of the EnterpriseDB Github Org, you'll want to follow these steps so that icons are pulled in the next steps.
+   1. Create a Github token which can pull private packages. [You can learn how to do that here.](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-to-github-packages)
```

Comment:
I think more explicit instructions is needed for PAT generation and usage:

* Specify the minimum scope needed: `read:packages`
* Enable SSO

etc.